apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/GrpcTraceConsumer.cs

```diff
@@ -32,18 +37,21 @@ internal GrpcTraceConsumer(Task<TraceServiceClient> client)
         }
 
         /// <inheritdoc />
-        public void Receive(Traces traces)
+        public void Receive(IEnumerable<TraceProto> traces)
         {
             GaxPreconditions.CheckNotNull(traces, nameof(traces));
 
+            TraceProto trace = traces.FirstOrDefault();
             // If their are no traces do not try to send them.
-            if (traces.Traces_.Count == 0)
+            if (trace == null)
             {
                 return;
             }
 
+            Traces tracesObj = new Traces();
```

Comment:
`Traces tracesObj = new Traces { Traces_ = { traces } };` should work fine. We designed protobuf generated code to work cleanly with object and collection initializers.